src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.trim() !== '');
```

Comment:
The current implementation will not handle cases where there are multiple semicolons without any styles between them, as it will create empty strings that are not filtered out. Consider using a regular expression to split the string by semicolons that are followed by a newline or the end of the string, ensuring that only valid style declarations are processed.